Tests/AbstractTests/BoundedSemilatticeTests.generated.swift

```diff
@@ -0,0 +1,48 @@
+// Generated using Sourcery 0.7.2 — https://github.com/krzysztofzablocki/Sourcery
```

Comment:
If the library is to be targeted also to Linux we cannot make that requirement because Sourcery is still not available for Linux. And in general I prefer to commit all the sources, autogenerated or not, so that the library works out-of-the-box, and in the end you don't really need to check the code in files  that start with, in this case `Generated using Sourcery`, but only the relative templates. But in this particular case code generation is only used for tests, so the library is still usable without Sourcery, which is needed only if you want to run tests locally.

If you think it's better to not commit the generated files is fine by me, but what about the Sourcery binary (and companion app)? Do you think we should add those to the git index?